IFC4x3/Constants/s/SUBSCRIBERSERVER_2XelPITJT1YeljviMUc_n2.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SUBSCRIBERSERVER_2XelPITJT1YeljviMUc_n2" Name="SUBSCRIBERSERVER" UniqueId="a1a2f652-7537-418a-8bed-e6c59e9bec42">
+	<Documentation>It is a database in charge of the management of mobile subscribers. It can be an authentication center (AuC)  or an home location register (HLR).</Documentation>
```

Comment:
```suggestion
	<Documentation>It is a database in charge of the management of mobile subscribers. It can be an authentication center (AuC)  or a home location register (HLR).</Documentation>
```